packages/dialog/src/vaadin-dialog.js

```diff
@@ -35,6 +39,7 @@ registerStyles(
 
     ::slotted([slot='title']) {
       font: inherit !important;
+      word-break: break-word;
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), `word-break: break-word` is deprecated, the alternative seems to be to use `overflow-wrap: anywhere`.

Reply:
Thanks, updated.